### ISSUE TITLE
On Windows 11 in Turkish, the on-screen keyboard does not type the letter 'i'.

### ISSUE THREAD

**Issue by orkuun** [musescore/MuseScore#33673]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

My computer runs the latest version of Windows 11 Home in Turkish and has a touchscreen with stylus support (Asus Vivobook 14 Flip). 
In MuseScore Studio version 4.7.2, when writing the 'title of the score,' 'composer name,' or 'lyrics' on the score, the on-screen keyboard (but only the on-screen keyboard) does not type the lowercase 'i' letter. 
My virtual keyboard layout is Turkish Q. By the way, there is no problem with the physical keyboard.
I checked in other applications and there is no such problem(word etc).

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/02c8a4cd-b9c5-45c3-9a7d-1519bb80a1f9

### In which versions of MuseScore Studio is this issue present?

4.5.2

### Regression

I was unable to check

### Operating system

windows 11

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
